layouts/BlogCategoryLayout.tsx

```diff
@@ -8,45 +8,56 @@ import Link from '@/components/Link';
 import Pagination from '@/components/Pagination';
 import getBlogData from '@/next-data/blogData';
 
-const getCurrentCategory = (pathname: string) => {
+const getCategoryData = async (pathname: string) => {
   // We split the pathname to retrieve the blog category from it since the
   // URL is usually /blog/{category} the second path piece is usually the
   // category name, which usually year-YYYY
   const [, _pathname, category] = pathname.split('/');
-
   if (_pathname === 'blog' && category && category.length) {
-    return category;
+    const data = await getBlogData(category);
```

Comment:
We prefer inline return objects: Can you change to:

```tsx
const data = await getBlogData(category);

return { ...data, category };
```

Reply:
Ah, I had assumed prettier would apply preferences like that